tests/compose/test_column_transformer.py

```diff
@@ -39,3 +42,45 @@ def test_column_transformer():
     expected = clone(a).fit_transform(df)
     expected = pd.DataFrame(expected, index=result.index, columns=result.columns)
     dd.utils.assert_eq(result, expected)
+
+
+def test_column_transformer_unk_chunksize():
+    cols = 3
+    rows = 4
+    a = 0x61  # character code for 'a'.
+    names = list(map(chr, a + np.arange(cols)))
```

Comment:
I think `names = ['a', 'b', 'c']` is a bit more readable.

Reply:
done